### DISCUSSION TITLE
DateRangePicker limit time selection to "00" or "30"

### DISCUSSION THREAD

**Discussion by choarau-craft** [rsuite/rsuite#3771] [Q&A]
Hi,

Is there anyway to limit time selection to a restricted list of values ?
In my case I'd like to use only [0, 30], but could be [0,15,30,45].

Thanks

**Comment by simonguo** [comment]
Hi  @choarau-craft.

DatePicker has the `hideHours/Minutes/Seconds` method, which can limit the value list of time. But the DateRangePicker does not support it yet, and it needs to be considered to add this feature.
